pkg/gadgets/utils/test/helpers.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/cilium/ebpf"
+	"github.com/google/go-cmp/cmp"
+)
+
+// CreateMntNsFilterMap creates and fills an eBPF map that can be used
+// to filter by mount namespace in the different tracers.
+func CreateMntNsFilterMap(mountnsids ...uint64) (*ebpf.Map, error) {
```

Comment:
If this is a test helper, how about taking *testing.T, calling t.Helper() and use t.Fatal() for error handling? That makes the target test code shorter.